docs/dev-installation.rst

```diff
@@ -0,0 +1,74 @@
+
+Developer Installation 
+====================== 
+ 
+You can either install Girder natively on you machine or inside a virtual
+machine with Vagrant. 
+ 
+Virtual Machine 
++++++++++++++++ 
+ 
+The easiest way to develop for girder is within a virtual machine using Vagrant.
+For this, you need `Vagrant <https://www.vagrantup.com/downloads.html>`_ and `VirtualBox <https://www.virtualbox.org/wiki/Downloads>`_. 
+Girder is tested to work seamlessly with Vagrant 2.0 and VirtualBox 5.1. 
+ 
+Once you have those installed, obtain the Girder source code by cloning the Git 
+repository on `GitHub <https://github.com>`_: :: 
```

Comment:
Since this is generated with Sphinx, we can ensure it [applies additional language-specific formatting to code blocks](http://www.sphinx-doc.org/en/stable/markup/code.html#). `.. code-block:: bash` would be best to use in several places here.